sec-keys/sigstore-trusted-root/sigstore-trusted-root-0_p20241010.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="trusted-root.json for dev-python/sigstore"
+HOMEPAGE="https://www.sigstore.dev/"
+SRC_URI="
+	https://dev.gentoo.org/~mgorny/dist/${P}.tar.xz
```

Comment:
Is there some documentation on how to create this tarball?

Reply:
Jinx!

Reply:
Right. I've basically packed `${HOME}`, as created by `src_test()`. I don't even have a clue if we need all these files.